test/exceptions_tests.cpp

```diff
@@ -0,0 +1,42 @@
+#include <gtest/gtest.h>
+
+#include <string>
+
+#include "cpr/exceptions.h"
+#include <cpr/cpr.h>
+
+using namespace cpr;
+
+// HTTP code generator
+Url url{"https://httpstat.us/"};
```

Comment:
Please do not use an external URL, since tests should not depend on an internet connection.
The best would be to add a modified case for this one:
https://github.com/libcpr/cpr/blob/eccea39e4b112a088ae665eda5854cc366f86128/test/httpServer.cpp#L30-L39
Where you pass the code, you want to receive as header.
https://github.com/libcpr/cpr/blob/eccea39e4b112a088ae665eda5854cc366f86128/test/httpServer.cpp#L193